tests/integrators/assembler_simple_mgmatrix_04.cc

```diff
@@ -203,14 +204,14 @@ int main()
 
   // FESystem<2> sys1(p0, 2, p1, 1);
   // FESystem<2> sys2(p0, 2, rt0, 1);
-  // FESystem<2> sys3(rt0, 1, p0, 2);
+  FESystem<2> sys3(rt0, 1, p0, 1);
   // FESystem<2> sys4(p1, 2, q2, 2);
   // FESystem<2> sys5(q2, 2, p1, 2);
```

Comment:
I just checked, these FESystems were copied from assembler_simple_mgmatrix_03 were the correct placement of blocks in the global matrix is tested. So that here we do not the symmetric cases anymore and we can leave out sys2 and sys5. So that we are down to 
  FESystem<2> sys1(p0, 2, p1, 1);
  FESystem<2> sys2(rt0, 1, p0, 1);
  FESystem<2> sys3(q2, 2, p1, 1);
which results in 230k. What do you say, still to big?